app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,27 @@
+module CommentsHelper
+  def comment_path(comment)
+    polymorphic_path(
+      commentable_resource_array(comment.commentable),
+      anchor: dom_id(comment)
+    )
+  end
+
+  def commentable_path(comment)
```

Comment:
looks like these to methods could be merged if you make `anchor` an optional argument?

Reply:
I'm not a fan of the name though, it makes it look like a Rails route, but it isn't.